.changeset/chilly-walls-join.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Hey, ive created a new PR since rebasing was gicing me issues on this branch. Here is the link please refer to this PR from now on. Thank you 

https://github.com/smartcontractkit/external-adapters-js/pull/3529